.github/file-filters.yml

```diff
@@ -16,17 +16,25 @@ yarn_lockfile: &yarn_lockfile
 eslint_config: &eslint_config
   - '.eslint*'
 
-frontend: &frontend
+# `frontend_src` filters on files that are frontend changes excluding
+# changes to the tests/ directory.
+# If you want to filter on *all* frontend files, use the `frontend_all` filter.
+frontend_src: &frontend_src
   - *yarn_lockfile
-  - *frontend_lintable
   - *eslint_config
+  - '!(test)/**/*.[tj]{s,sx}'
```

Comment:
I think this accidentally excludes some things -- also looks like the new testsuite uses `.spec.{ext}` -- 

```console
$ git ls-files -- '*.tsx' '*.jsx' | grep test
static/app/stores/latestContextStore.tsx
static/app/utils/__mocks__/withLatestContext.tsx
static/app/utils/replays/useReplayData.test.jsx
static/app/utils/testableTransition.tsx
static/app/utils/withLatestContext.tsx
static/app/views/projectDetail/projectLatestAlerts.spec.jsx
static/app/views/projectDetail/projectLatestAlerts.tsx
static/app/views/projectDetail/projectLatestReleases.spec.jsx
static/app/views/projectDetail/projectLatestReleases.tsx
static/app/views/settings/project/server-side-sampling/testUtils.tsx
tests/js/sentry-test/charts.jsx
tests/js/sentry-test/createListeners.tsx
tests/js/sentry-test/dropdownMenu.tsx
tests/js/sentry-test/index.jsx
tests/js/sentry-test/initializeOrg.jsx
tests/js/sentry-test/mockRouterPush.jsx
tests/js/sentry-test/modal.jsx
tests/js/sentry-test/profiling/utils.tsx
tests/js/sentry-test/providers/breadcrumbContextProvider.tsx
tests/js/sentry-test/reactTestingLibrary.spec.tsx
tests/js/sentry-test/reactTestingLibrary.tsx
tests/js/sentry-test/utils.tsx
```

Reply:
(After fixing my typo of `test` to `tests`) This will exclude tests from being classed as `frontend_src` but `static/` will continue to be included in `frontend_src`.

The `.spec` prefix isn't a consideration, the matching is just the prefix and the top level directory.

You can see which filters trigger which labels in this run: https://github.com/getsentry/sentry/actions/runs/3084072292/jobs/4985777928

```
2022-09-19T16:35:06.4784762Z ##[group]Filter frontend_src = true
2022-09-19T16:35:06.4784966Z Matching files:
2022-09-19T16:35:06.4785305Z src/sentry/static/sentry/vendor/aui-prototyping.9.1.5.js [modified]
2022-09-19T16:35:06.4785599Z static/app/utils/fields/index.ts [modified]
2022-09-19T16:35:06.4785900Z static/app/views/acceptOrganizationInvite.spec.jsx [modified]
2022-09-19T16:35:06.4786238Z static/app/views/userFeedback/userFeedbackEmpty.spec.tsx [modified]
2022-09-19T16:35:06.4786664Z static/app/views/userFeedback/userFeedbackEmpty.tsx [modified]
2022-09-19T16:35:06.4787777Z ##[endgroup]
2022-09-19T16:35:06.4788112Z ##[group]Filter frontend_all = true
2022-09-19T16:35:06.4788318Z Matching files:
2022-09-19T16:35:06.4788647Z src/sentry/static/sentry/vendor/aui-prototyping.9.1.5.js [modified]
2022-09-19T16:35:06.4788937Z static/app/utils/fields/index.ts [modified]
2022-09-19T16:35:06.4789239Z static/app/views/acceptOrganizationInvite.spec.jsx [modified]
2022-09-19T16:35:06.4789572Z static/app/views/userFeedback/userFeedbackEmpty.spec.tsx [modified]
2022-09-19T16:35:06.4789903Z static/app/views/userFeedback/userFeedbackEmpty.tsx [modified]
2022-09-19T16:35:06.4790202Z tests/js/instrumentedEnv/index.js [modified]
2022-09-19T16:35:06.4790492Z tests/js/instrumentedEnv/userEventIntegration.ts [modified]
2022-09-19T16:35:06.4790843Z tests/js/sentry-test/dropdownMenu.tsx [modified]
2022-09-19T16:35:06.4791164Z tests/js/sentry-test/modal.jsx [modified]
```